include/mega/megaclient.h

```diff
@@ -77,15 +77,12 @@ class MEGA_API MegaClient
     void confirmsignuplink(const byte*, unsigned, uint64_t);
     void setkeypair();
 
-    /**
-     * @brief Initialises the Ed25519 EdDSA key user properties.
-     *
-     * A key pair will be added, if not present, yet.
-     *
-     * @return Error code (default: 1 on success).
-     */
+    // initialises the Ed25519 EdDSA key user properties
```

Comment:
Doxygen string was duplicated, both in header and source file. For readability, we keep a different style of comments in header files, so I kept your doxygen string in source file.
Anyway, this function does not accept any parameter :)